### ISSUE TITLE
NTR: sickle foot

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#10621]
**Preferred term label:**
sickle foot

**Synonyms**
sickled feet, metatarsus adductus

**Definition (free text, please give PubMed ID)**
The toes point inward, resulting in a sickle shape of the foot. Sickle foot is a common, usually congenital foot deformity whose cause is unclear.

**Parent term (use hpo.jax.org/app)**
Positional foot deformity HP:0005656

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0003-0113-912X, 0000-0002-7371-8158

**Comment by pnrobinson** [comment]
@GrieseAG 
This was [Metatarsus adductus HP:0001840](https://hpo.jax.org/browse/term/HP:0001840)
I will add the synonyms.

**Comment by pnrobinson** [comment]
done!
